### PR TITLE
add Loop Drop app

### PR DESCRIPTION
**Pull request by mattdesl**
it's awesome :tada:

https://www.youtube.com/watch?v=EBkmdNDIR6E

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Some good apps written with Electron.
 - [Sample apps](https://github.com/hokein/electron-sample-apps) - Various sample apps.
 - [Koko](https://github.com/hachibasu/koko) - IRC client.
 - [Leanote](https://github.com/leanote/desktop-app) - Cloud notepad.
+- [Loop Drop](https://github.com/mmckegg/loop-drop-app) - Live electronic music performance software.
```
Comment (sindresorhus) [inline readme.md]: Should be added to the bottom.

Can you come up with a description without `software`?

**Comment by sindresorhus** [comment]
Looks cool :)

**Comment by mattdesl** [comment]
> Should be added to the bottom. 

The bottom of the list? Isn't it alphabetical?

Updated description.

**Comment by sindresorhus** [comment]
> The bottom of the list? Isn't it alphabetical?

It's clearly not. Look at it.

**Comment by mattdesl** [comment]
> It's clearly not. Look at it.

A simple "nope" would have sufficed... :unamused: 

Anyways, updated.

**Comment by arthurvr** [comment]
Please fix the merge conflict.

**Comment by sindresorhus** [comment]
> A simple "nope" would have sufficed...

Sorry if it sounded rude. I just think you could have easily checked it yourself instead of adding another unnecessary back and forth. Anyways, thanks for the contribution :)

**Pull request closed without merging**